app/scripts/lib/session.js

```diff
@@ -41,7 +41,22 @@ define([
         // ignore the parse error.
       }
 
-      this.set(values);
+      // Update new values, without overwriting items on the prototype.
```

Comment:
no, this is more about just sucking in any changes that were made to the backing store